content/applications/productivity/knowledge/management.rst

```diff
@@ -1,114 +1,209 @@
+#. Click :guilabel:`Share` in the top-right menu.
+#. Click :guilabel:`Invite`.
+#. Select the preferred :guilabel:`Permission` and add users in the :guilabel:`Recipients` field.
+#. Click :guilabel:`Invite`.
 
-.. _management/categories:
+.. _knowledge/management/share-online:
 
-Categories
-----------
+Generate article URL
+--------------------
 
-Additionally, articles are divided into four categories that can be found on the left sidebar. These
-categories give articles default **access rights**.
+Click :guilabel:`Share` and activate the :guilabel:`Share to web` toggle to generate a URL.
+Click the :icon:`fa-clone` (:guilabel:`copy`) icon to copy the article's URL.
 
-- :guilabel:`Favorites`: You can set any article you can access as a favorite. To do so, click the
-  star-shaped icon (:guilabel:`★`) in the top-right menu of articles. Marking articles as
-  **favorites** is user-specific and does not affect other users.
-- :guilabel:`Workspace`: Articles displayed in that category are available to all internal users.
-  These users have the right to read, modify or share these articles.
-- :guilabel:`Shared`: Articles displayed in that category are those you shared with internal users,
-  external users, or shared with you.
-- :guilabel:`Private`: Articles displayed in that category are only available to you.
+.. note::
+   - If an article contains :ref:`inserted views <knowledge/articles_editing/views>`, users with
+     the URL do not see them unless they can access the inserted content.
+   - Having the Website app is necessary to share an article's URL.
 
-.. image:: management/left-sidebar-cat.png
-   :alt: Categories displayed in the left sidebar of Odoo Knowledge
+.. _knowledge/management/remove:
+
+Removal
+=======
+
+Removing an article involves deleting or archiving it.
+
+Delete an article
+-----------------
+
+Select an article in the sidebar tree and click the :icon:`fa-ellipsis-v` (:guilabel:`ellipsis`)
+icon, then :guilabel:`Send to Trash`. The article is moved to the trash for 30 days before being
+permanently deleted.
+
+Alternatively, click :guilabel:`Search` in the top-left menu. Select an article and click
+:menuselection:`Actions --> Delete --> Delete` to remove the article permanently.
+
+Archive an article
+------------------
+
+Click :guilabel:`Search`, select an article, and click :menuselection:`Actions --> Archive -->
+Archive`.
+
+.. tip::
+   - To restore a deleted article, click :guilabel:`Open the Trash` at the bottom of the sidebar
+     tree, select an article, and click :guilabel:`Restore`. Alternatively, click :guilabel:`Search`
+     in the top-left menu. In the search bar, click :menuselection:`Filters --> Trashed`. Click the
+     article, then :guilabel:`Restore`.
+   - To restore an archived article, click :guilabel:`Search`. In the search bar, click
+     :menuselection:`Filters --> Archived`. Select an article and go to :menuselection:`Action -->
+     Unarchive`.
```

Comment:
I would put the first bullet point that info right after the info on deletion, and not necessarily as a tip.